code/modules/admin/topic.dm

```diff
@@ -11,9 +11,29 @@
 		return TRUE
 	log_admin_private("[key_name(usr)] clicked an href with [msg] authorization key! [href]")
 
-/datum/admins/Topic(href, href_list)
-	..()
+// BASE ADMINS_TOPIC
+
+/datum/admins_topic()

```

Comment:
Not sure what parenthesis are doing here.

Reply:
probably a mistake my end. I was doing some of this on a busy day where I kept getting pulled away.